packages/markers/src/browser/marker-tree-model.ts

```diff
@@ -1,9 +1,9 @@
 /*
-* Copyright (C) 2017 TypeFox and others.
-*
-* Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
-*/
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
ditto not needed

Reply:
he fixed the indentation, that he broke in a previous commit. No big deal but ok to do it here, no?

Reply:
I fixed an erroneous indentation here which I've pushed accidentally before.

Reply:
OK I had not noticed